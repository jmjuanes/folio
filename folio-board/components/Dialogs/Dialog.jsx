import React from "react";

export const Dialog = props => (
    <div
        className={props.className}
        style={{
            position: "absolute",
            ...props.style,
        }}
    >
        <div
            className="bg:white r:xl shadow:md p:6 z:5 b:1 b:solid b:light-900"
            style={{
                width: props.width,
            }}
        >
            {props.children}
        </div>
    </div>
);

Dialog.defaultProps = {
    className: "",
    width: "16rem",
    style: {},
};
