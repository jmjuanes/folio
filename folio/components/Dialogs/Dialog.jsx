import React from "react";

export const Dialog = props => (
    <div className={props.className} style={{position: "absolute", ...props.style}}>
        <div className="b-1 b-solid b-gray-900 bg-white r-xl shadow-md p-4" style={{width: props.width}}>
            {props.children}
        </div>
    </div>
);

Dialog.defaultProps = {
    className: "",
    // active: false,
    width: "15rem",
    style: {},
};
