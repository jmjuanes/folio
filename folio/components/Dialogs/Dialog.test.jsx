import {create} from "react-test-renderer";

import {Dialog} from "./Dialog.jsx";

describe("Dialog", () => {
    it("should render", () => {
        const component = create(<Dialog />);

        expect(component.toJSON).toMatchSnapshot();
    });
});
