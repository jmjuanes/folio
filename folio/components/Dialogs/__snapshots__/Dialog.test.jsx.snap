// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog should render 1`] = `[Function]`;
